infer/src/IR/Procdesc.mli

```diff
@@ -315,6 +317,8 @@ val fold_nodes : t -> init:'accum -> f:('accum -> Node.t -> 'accum) -> 'accum
 val fold_slope_range : Node.t -> Node.t -> init:'accum -> f:('accum -> Node.t -> 'accum) -> 'accum
 (** fold between two nodes or until we reach a branching structure *)
 
+val set_succs_exn_base : Node.t -> Node.t list -> Node.t list -> unit
```

Comment:
> Why not use `set_succs` instead?
> 
> If this is needed could you add a doc string explaining that it should be used carefully because it won't update the node's successors' pred links? (not sure why that is ok except when the current node is known to have no succs, but then this is the same as `set_succs` just below) Named arguments like `set_succs` would be nice too since the types don't help to disambiguate the arguments here.

It was an outdated function. I have replaced it with `set_succs`